app/models/changeset/commit.rb

```diff
@@ -38,7 +32,7 @@ def short_sha
 
   # @return [Integer, NilClass]
   def pull_request_number
-    if number = summary[PULL_REQUEST_MERGE_MESSAGE, 1] || summary[PULL_REQUEST_SQUASH_MESSAGE, 1]
+    if number = @data.commit.message[PULL_REQUEST_MESSAGE, 1]
```

Comment:
truncate was weird, but we should match in the first line

Reply:
Yup, you're right 😊
Sorted now 👍